submissions/Roka20012/js-dom/interactive.js

```diff
@@ -53,8 +54,8 @@ function showStory(index) {
 function checkStory({ target }) {
 
     if (target.tagName === "A") {//перевіряємо чи елемент на який ми натиснули це пункт меню
```

Comment:
I hurried with tagName :D It's better to use `closest` method [read here](https://developer.mozilla.org/en-US/docs/Web/API/Element/closest) with event delegation. in this case would be something like this if(target.closest('#navbar') {..}. That is about more universal code for your markup, to avoid an influance of minor changes in your style on your code.

Reply:
Thanks i fix it